python/proton/_data.py

```diff
@@ -686,7 +686,8 @@ def _check(self, err):
             exc = EXCEPTIONS.get(err, DataException)
             raise exc("[%s]: %s" % (
                 err,
-                ffi.string(pn_error_text(pn_data_error(self._data))).decode('utf8')
+                # ffi.string(pn_error_text(pn_data_error(self._data))).decode('utf8')
+                pn_error_text(pn_data_error(self._data))
```

Comment:
IMO the `.decode()` version looks more likely to be correct here, but I am not sure from just looking at it. I guess you had a reason for the change.

Reply:
I commented it purpose of debugging the null pointer issue. It should be left with .decode() for clear error message